Library/Formula/gpg-agent.rb

```diff
@@ -33,9 +33,46 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+      To replace ssh-agent with gpg-agent, you need to:
+         * ensure that "enable-ssh-support" is in ~/.gnupg/gpg-agent.conf
+         * ensure that "pinentry" in ~/.gnupg/gpg-agent.conf has a graphical PIN entry
+           such as pinentry-mac (needed for GUI programs)
+         * disable the system ssh-agent with
+             launchctl unload -w /System/Library/LaunchAgents/org.openssh.ssh-agent.plist
+         * have launchd start gpg-agent at login
+    EOS
+  end
```

Comment:
I'll admit I've only used OSX from 10.9 on, but that plist has been present and running on every Mac I've ever touched, including a freshly reinstalled mini.  It's kind of a fun one with how it starts though, because the org.openbsd.ssh-agent.plist uses the SecureSocketWithKey listener, the environmental variable gets set but ssh-agent isn't actually started until launchd detects action on the socket.

Oh, it does look like I called the file wrong, though, it's org.openbsd.ssh-agent.plist, which def. needs fixing.

I can wrap it in the conditional, I'm just fairly certain (without having a 10.8 machine handy) that that's a default plist that's live and active on all fresh installs.